### ISSUE TITLE
docker run Conflicting options: --rm and -d 

### ISSUE THREAD

**Issue by trishnaguha** [docker/docker#27812]
The command is:
```
docker run --rm --name helloworld -d -p 5000:5000 fedora/flask-app:test
```
``-rm`` and ``-d`` in ``docker run`` command should not be conflicting.

The error log:
```
TASK [Run Docker Container] ****************************************************
fatal: [209.132.184.95]: FAILED! => {"changed": true, "cmd": ["docker", "run", "--rm", "--name", "helloworld", "-d", "-p", "5000:5000", "fedora/flask-app:test"], "delta": "0:00:00.087692", "end": "2016-10-27 06:10:04.793428", "failed": true, "rc": 1, "start": "2016-10-27 06:10:04.705736", "stderr": "Conflicting options: --rm and -d", "stdout": "", "stdout_lines": [], "warnings": []}
```

Here is more info on Docker installed:
```
[fedora@app-deploy-test ~]$ docker --version
Docker version 1.10.3, build a612434/1.10.3
[fedora@app-deploy-test ~]$ sudo docker info
Containers: 1
 Running: 0
 Paused: 0
 Stopped: 1
Images: 21
Server Version: 1.10.3
Storage Driver: devicemapper
 Pool Name: atomicos-docker--pool
 Pool Blocksize: 524.3 kB
 Base Device Size: 10.74 GB
 Backing Filesystem: xfs
 Data file: 
 Metadata file: 
 Data Space Used: 1.844 GB
 Data Space Total: 15.77 GB
 Data Space Available: 13.93 GB
 Metadata Space Used: 413.7 kB
 Metadata Space Total: 46.14 MB
 Metadata Space Available: 45.72 MB
 Udev Sync Supported: true
 Deferred Removal Enabled: true
 Deferred Deletion Enabled: true
 Deferred Deleted Device Count: 0
 Library Version: 1.02.122 (2016-04-09)
Execution Driver: native-0.2
Logging Driver: journald
Plugins: 
 Volume: local
 Network: host bridge null
Kernel Version: 4.7.3-200.fc24.x86_64
Operating System: Fedora 24 (Atomic Host)
OSType: linux
Architecture: x86_64
Number of Docker Hooks: 2
CPUs: 2
Total Memory: 3.86 GiB
Name: app-deploy-test.novalocal
ID: X5RC:XOCC:3GMD:WGKB:7IZ5:RU7B:ZLPI:OG2X:3AOK:ROOD:VJGK:EWJI
Registries: docker.io (secure)
```

**Comment by shaynewang** [comment]
Issue was addressed here: [https://github.com/docker/docker/issues/6003]
This conflict will be fixed in the 1.13 release[https://github.com/docker/docker/pull/20848]

**Comment by trishnaguha** [comment]
Ah thanks! closing this one.
